data/licenses/LICENSE.bulk-extractor

```diff
@@ -0,0 +1,44 @@
+Public Domain Software
```

Comment:
Upstream has updated license to MIT please update this file
https://github.com/simsong/bulk_extractor/blob/master/LICENSE.md

Reply:
License updated.